src/main/handlebars/group.handlebars

```diff
@@ -8,17 +8,17 @@
         </div>
       {{/if}}
     </div>
-    <p id="description">
+    <div id="description">
       {{#with group}}
-        {{#*fragment "description"}}{{description}}{{/fragment}}
+        {{#*fragment "description"}}{{& description}}{{/fragment}}
       {{/with}}
-    </p>
+    </div>
     <hr>
 
     <form hx-post="/group/{{group._id}}/posts" hx-target="#posts" hx-swap="afterbegin" hx-on::after-request="this.reset()">
       <div>
         <label for="body">What's on your mind?</label>
-        <textarea name="body" cols="80" rows="4"></textarea>
+        {{> components/editor id="post" field="body"}}
```

Comment:
Is *components* good naming here? These are different than e.g. React or Vue components, and certainly differ from web components.

https://www.merriam-webster.com/thesaurus/component